docs/usage.rst

```diff
@@ -0,0 +1,197 @@
+Naggum usage
+============
+
+Currently there are two dialects of Naggum: high level *Compiler* and low-level
+*Assembler*.
+
+Naggum Compiler
+---------------
+
+Command line syntax for Naggum Compiler is::
+
+    $ Naggum.Compiler [source1.naggum] [source2.naggum] [/r:assembly1] [/r:assembly2]
+
+Each input source file will be compiled to a separate executable assembly (i.e.
+an ``.exe`` file) in the current directory.
+
+``.naggum`` extension is recommended for high level Naggum files.
+
+Naggum Assembler
+----------------
+
+Naggum Assembler uses low level Naggum dialect. Command line syntax is::
+
+    $ Naggum.Assembler [source1.nga] [source2.nga]
+
+Assembler output is determined based on input file content: each file may
```

Comment:
I feel like you meant "depends", not "determined based", but I'm not totally sure of that.

I also don't understand why you even wrote this sentence. If a person doesn't understand that the output depends on the input, maybe they should take an introductory programming class first?